src/interface/src/styleguide/scenario-card/scenario-card.component.ts

```diff
@@ -20,7 +20,20 @@ import { ButtonComponent } from '../button/button.component';
 import { MatIconModule } from '@angular/material/icon';
 import { MatMenuModule } from '@angular/material/menu';
 import { MatButtonModule } from '@angular/material/button';
-export type ScenarioStatus = 'Running' | 'Done' | 'Failed';
+
+export type ScenarioResultLabel = 'Done' | 'Running' | 'Failed';
+
+// redefining this works, but including it kills storybook?
+export type ScenarioResultStatus =
```

Comment:
Note: There's something related to Leaflet that breaks storybook when I try to include ScenarioResultStatus from `@types` here. Needs more investigation.